packages/server/graphql/mutations/addComment.ts

```diff
@@ -89,6 +139,21 @@ const addComment = {
       }
     }
 
+    const notificationsToAdd = getMentionNotifications(
```

Comment:
-1 what about updating/deleting comments? if i add/remove a mention in an update, should we diff them to make sure new mentionees get a notification?

Reply:
We probably should, but the edge cases might get a bit hairy - e.g. if a user is mentioned, unmentioned, and mentioned again, we'd only want one notification. I think keeping it simple for now is the way to go, but I added a TODO to `updateCommentContent`.

Reply:
yep, i think GitHub does a good job of managing that whenever you @-mention someone in some text. happy to merge to keep the PRs small, I'd encourage adding that feature though. adding someone in an edit & not having that turn into a notification would feel like a bug

Reply:
That's how Slack does it by the way. If you @-mention in an edit, then no notification is sent.

Reply:
whoa! TIL. makes sense though, more ephemeral thing = created faster = no notification. something longer lasting like a GH issue = longer lasting = notification.